cpp/include/nvtext/byte_pair_encoding.hpp

```diff
@@ -32,11 +32,11 @@ namespace nvtext {
 /**
  * @brief The table of merge pairs for the BPE encoder.
  *
- * To create an instance, call @ref nvtext::load_merge_pairs_file
+ * To create an instance, call @ref nvtext::load_merge_pairs
  */
 struct bpe_merge_pairs {
   struct bpe_merge_pairs_impl;
-  std::unique_ptr<bpe_merge_pairs_impl> impl{};  ///< Implementation of the BPE merge pairs table.
+  bpe_merge_pairs_impl* impl{};  ///< Implementation of the BPE merge pairs table.
```

Comment:
So this will no longer manage the pointer which may cause memory leak. Is it intentional?

Reply:
Ah, good catch. I meant for the destructor to free the pointer. I need to change this back.